colossalai/zero/sharded_model/sharded_model_v2.py

```diff
@@ -91,6 +93,20 @@ def backward_by_grad(self, tensor, grad):
         torch.autograd.backward(tensors=tensor, grad_tensors=grad)
         self._final_backward_hook()
 
+    def deepcopy_to(self, other_model: nn.Module):
```

Comment:
I feel this method is more like a util method in the unit test module instead of an api of sharded model which can be called by the user.

Reply:
good idea. DONE